src/styles/index.scss

```diff
@@ -1,3 +1,47 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/header';
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+a,
+p {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Again, according to the BEM methodology, it's not recommended to use tags for styling. Please, create classes for 'a' and 'p' tags and apply the styles to these classes.